apps/dog-food/Main.hs

```diff
@@ -122,18 +126,24 @@ chatHelpInfo =
   map
     styleMarkdown
     [ "Using chat:",
-      highlight "/add <name>" <> "       - create invitation to send out-of-band",
-      "                    to your contact <name>",
-      "                    (any unique string without spaces)",
-      highlight "/accept <name> <invitation>" <> " - accept <invitation>",
-      "                    (a string that starts from \"smp::\")",
-      "                    from your contact <name>",
-      highlight "/name <name>" <> "      - set <name> to use in invitations",
+      highlight "/add <name>" <> "       - create invitation to send out-of-band to your contact <name>",
+      "                    (<name> is the alias you choose to message your contact)",
+      highlight "/connect <name> <invitation>" <> " - connect using <invitation>",
+      "                    (a string returned by /add that starts from \"smp::\")",
+      "                    if /connect is used by your contact,",
+      "                    <name> is the alias your contact chooses to message you",
```

Comment:
kind of strange to explain this from a counterparty point of view - you also connect via somebody's else invitations, maybe
`<name> is the alias you choose to message the contact who invited you (your inviter??)`

Reply:
yes, the whole thing needs improvement...